Modules/Core/Tests/Feature/ImportControllerTest.php

```
@@ -35,4 +37,368 @@ public function it_displays_import_page(): void
+        ];
+
+        /** Act */
+        $this->actingAs($user);
+        $response = $this->post(route('import.form'), $formData);
+
+        /** Assert */
+        $response->assertRedirect(route('import.index'));
+        
+        // Cleanup
+        @unlink($uploadDir . '/clients.csv');
+    }
```

PHPMD (2.15.0)

Remove error control operator '@' on line 201. (undefined)

(ErrorControlOperator)